pkg/operator/ceph/nfs/config.go

```diff
@@ -117,19 +117,24 @@ NFSv4 {
 }
 
 RADOS_KV {
-	ceph_conf = '` + cephclient.DefaultConfigFilePath() + `';
+	ceph_conf = "` + cephclient.DefaultConfigFilePath() + `";
 	userid = ` + userID + `;
 	nodeid = ` + nodeID + `;
 	pool = "` + n.Spec.RADOS.Pool + `";
 	namespace = "` + n.Spec.RADOS.Namespace + `";
 }
 
 RADOS_URLS {
-	ceph_conf = '` + cephclient.DefaultConfigFilePath() + `';
+	ceph_conf = "` + cephclient.DefaultConfigFilePath() + `";
 	userid = ` + userID + `;
 	watch_url = '` + url + `';
 }
 
+RGW {
+        name = "client.` + userID + `";
```

Comment:
From the ganesha docs (https://github.com/ceph/ceph/blob/main/doc/radosgw/nfs.rst) this is supposed to be the RGW instance name. In your testing, did this still work for you? It seems to me that it shouldn't work. Did you have to do manual configuration to allow this to work?

It may be good to add a section to the NFS docs detailing how to manually attach it to RGW. It would be good to add a block noting it as experimental.

```
!!! warning
    Experimental
```

Reply:
yes, with manual configuration, it works well with the addition. In my environment, instance name should be prefixed with "client." to be forming a valid auth entity name for this to work. the nfs doc shows it would be client.rgw, which looks to be a bit variation off the fact.

Reply:
The docs definitely need to be updated to detail how to attach to RGW with a the experimental warning.